### PR TITLE
WIP - Moojjoo patch 1

### PR DESCRIPTION
**Pull request by moojjoo**
Robert "Moojjoo" Dannelly Notes from MSFT Premier Ticket
Associate the AzureAD trusted identity token issuer to the appropriate web application and zone
Configure AzureCP 
a.	Remove the default Mail claim mapping 
b.	Change the property to look users up by for the email address claim from UPN to Mail. 
    i.	Display the mail property and set the picker entity type to Email
Configure the Claim configuration in AzureAD for your On-premises App to use the Mail property as the Name claim and format for the nameidentifier

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 1]
```diff
@@ -107,17 +107,17 @@ Copy the *Identifier* value into the *Realm* property into a table  (See Table 1
...
 > Replace the */saml2* value in the URL with */wsfed*. The */saml2* endpoint will process SAML 2.0 tokens. The */wsfed* endpoint enables processing SAML 1.1 tokens and is required for SharePoint 2016 SAML federation.
 
-## Step 4: Configure a new trusted identity provider in SharePoint Server 2016
+## Step 4: Configure a new trusted identity provider in SharePoint Server 2013 or 2016
 
-Sign into the SharePoint Server 2016 server and open the SharePoint 2016 Management Shell. Fill in the values of $realm, $wsfedurl, and $filepath from Table 1 and run the following commands to configure a new trusted identity provider.
+Sign into the SharePoint Server 2016 server and open the SharePoint 2013 or 2016 Management Shell. Fill in the values of $realm, $wsfedurl, and $filepath from Table 1 and run the following commands to configure a new trusted identity provider.
 
 > [!TIP]
 > If you're new to using PowerShell or want to learn more about how PowerShell works, see [SharePoint PowerShell](https://docs.microsoft.com/en-us/powershell/sharepoint/overview?view=sharepoint-ps).
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Suggestion (not part of the changed material but necessary to remove en-us from links)
**https://docs.microsoft.com/powershell/sharepoint/overview?view=sharepoint-ps**

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 2]
```diff
@@ -117,7 +121,7 @@ Copy the *Identifier* value into the *Realm* property into a table  (See Table 1
 
 ## Step 4: Configure a new trusted identity provider in SharePoint Server 2016
 
-Sign into the SharePoint Server 2016 server and open the SharePoint 2016 Management Shell. Fill in the values of $realm, $wsfedurl, and $filepath from Table 1 and run the following commands to configure a new trusted identity provider.
+Sign into the SharePoint Server 2016 server and open the 2016 Management Shell. Fill in the values of $realm, $wsfedurl, and $filepath from Table 1 and run the following commands to configure a new trusted identity provider.
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Suggest these terms be in italics: 
Fill in the values of **$realm**, **$wsfedurl**, and **$filepath** from Table 1

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 3]
```diff
@@ -74,7 +74,11 @@ This article was written using the reference architecture in [Run a high availab
...
 
-1. Go to **Central Administration** > **Application Management** > **Manage Web Applications**, and choose the web application that needs to be extended to use SSL. Select the web application and click the **Extend ribbon** button. Extend the web application to use the same URL but use SSL with port 443.</br>![Extending the web app to another IIS site](images/SAML11/fig3-extendwebapptoiis.png)</br>
+1. Go to **Central Administration** > **Application Management** > **Manage Web Applications**, and choose the web application that needs to be extended to use SSL. Select the web application and click the **Extend ribbon** button. Extend the web application to use the same URL but use SSL with port 443. 
+
+> [!NOTE]
+> Very Important to EXTEND the EXISITING APPLICATION to a NEW ZONE and add Custom Sign In Page /_trust/.
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: **It is VERY IMPORTANT** to **extend** the **existing application** to a **new zone** and add a **Custom sign in page.**

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 4]
```diff
@@ -231,6 +237,15 @@ There is no validation on the values you search for, which can lead to misspelli
 
 To assist with this scenario, there is an open-source solution called [AzureCP](https://yvand.github.io/AzureCP/) that provides a custom claims provider for SharePoint 2016. It will use the Azure AD Graph to resolve what users enter and perform validation. Learn more at [AzureCP](https://yvand.github.io/AzureCP/). 
 
+## Robert "Moojjoo" Dannelly Notes from MSFT Premier Ticket
+1.	Associate the AzureAD trusted identity token issuer to the appropriate web application and zone
+2.	Configure AzureCP 
+a.	Remove the default Mail claim mapping
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Remove the default **m**ail claim mapping

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 5]
```diff
@@ -231,6 +237,15 @@ There is no validation on the values you search for, which can lead to misspelli
 
 To assist with this scenario, there is an open-source solution called [AzureCP](https://yvand.github.io/AzureCP/) that provides a custom claims provider for SharePoint 2016. It will use the Azure AD Graph to resolve what users enter and perform validation. Learn more at [AzureCP](https://yvand.github.io/AzureCP/). 
 
+## Robert "Moojjoo" Dannelly Notes from MSFT Premier Ticket
+1.	Associate the AzureAD trusted identity token issuer to the appropriate web application and zone
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Azure AD

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 6]
```diff
@@ -231,6 +237,15 @@ There is no validation on the values you search for, which can lead to misspelli
...
+2.	Configure AzureCP 
+a.	Remove the default Mail claim mapping
+b.	Change the property to look users up by for the email address claim from UPN to Mail.
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Change the property **used** to look users up for the email address claim from **UPN** to **Mail**.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 7]
```diff
@@ -231,6 +237,15 @@ There is no validation on the values you search for, which can lead to misspelli
 
 To assist with this scenario, there is an open-source solution called [AzureCP](https://yvand.github.io/AzureCP/) that provides a custom claims provider for SharePoint 2016. It will use the Azure AD Graph to resolve what users enter and perform validation. Learn more at [AzureCP](https://yvand.github.io/AzureCP/). 
 
+## Robert "Moojjoo" Dannelly Notes from MSFT Premier Ticket
+1.	Associate the AzureAD trusted identity token issuer to the appropriate web application and zone
+2.	Configure AzureCP 
+a.	Remove the default Mail claim mapping
+b.	Change the property to look users up by for the email address claim from UPN to Mail. 
+i.	Display the mail property and set the picker entity type to Email
+3.	Configure the Claim configuration in AzureAD for your On-premises App to use the Mail property as the Name claim and format for the nameidentifier
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Configure the **c**laim configuration in **Azure AD** for your **o**n-premises app to use the mail property as the **n**ame claim and format for the **nameidentifier**

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md [hunk 8]
```diff
@@ -231,6 +237,15 @@ There is no validation on the values you search for, which can lead to misspelli
 
 To assist with this scenario, there is an open-source solution called [AzureCP](https://yvand.github.io/AzureCP/) that provides a custom claims provider for SharePoint 2016. It will use the Azure AD Graph to resolve what users enter and perform validation. Learn more at [AzureCP](https://yvand.github.io/AzureCP/). 
 
+## Robert "Moojjoo" Dannelly Notes from MSFT Premier Ticket
+1.	Associate the AzureAD trusted identity token issuer to the appropriate web application and zone
+2.	Configure AzureCP 
+a.	Remove the default Mail claim mapping
+b.	Change the property to look users up by for the email address claim from UPN to Mail. 
+i.	Display the mail property and set the picker entity type to Email
```
Comment (nenonix) [inline Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: (Remove periods from all sentences or add to all in this list, and bold names or terms)

Display the mail property and set the picker entity type to **Email**

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 1 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Thank you nenonix.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 2 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Looks good :)

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 5 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Agree.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 6 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Check, great review.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 7 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Thank you.  Will keep this in mind in the future.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 4 above)
Comment (moojjoo) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Thank you for the edits.  👍

**Comment by JohanFreelancer9** [comment]
Copy reviewed - no further changes.

File: Enterprise/using-azure-ad-for-sharepoint-server-authentication.md (hunk 5 above)
Comment (nenonix) [reply Enterprise/using-azure-ad-for-sharepoint-server-authentication.md]: Thanks for being so receptive. A word of warning, I'm not always right, the real experts are the Microsoft owners and authors and they will have final word.

**Comment by moojjoo** [comment]
> @MSFTTracyP - Can you please assist to check, approve and merge the content for this PR?
> 
> Thank you.
myPil, thank you for reviewing this.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR?

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR?

Thank you.

**Comment by MSFTTracyP** [comment]
This documentation is set to be retired and people redirected to https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/sharepoint-on-premises-tutorial. Please find out if the edits at the start of this thread (from an MSFT Prem ticket) exist in the Azure article. If not, please comment in the thread to let me know. If the edits are too extensive, we can merge and ask the Azure team to please be sure the information in their copy is updated with required info. I'm waiting on an ETA for retirement and redirection. I appreciate the effort on this, everyone! With hope we get it sorted for customers soon!

**Comment by moojjoo** [comment]
> This documentation is set to be retired and people redirected to https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/sharepoint-on-premises-tutorial. Please find out if the edits at the start of this thread (from an MSFT Prem ticket) exist in the Azure article. If not, please comment in the thread to let me know. If the edits are too extensive, we can merge and ask the Azure team to please be sure the information in their copy is updated with required info. I'm waiting on an ETA for retirement and redirection. I appreciate the effort on this, everyone! With hope we get it sorted for customers soon!

@MSFTTracyP, just some additional information, the Premier Ticket came from working with MSFT MAJOR Customer in the Energy | Utility sector in the SouthEast (Little research you can probably figure out the customer 👍 ).  It was critical to provide a solution for outside contractors access (aka in the field construction workers) to communicate with internal SharePoint Sites.  It will be a shame to see that after many hours of research by your Support Level III engineer, Yvan (AzureCP), and developers who have always supported Microsoft as a solution to see this great piece of documentation retired.  I have been an evangelist | developer | engineer | supporter of Microsoft since MS-DOS and in 2003 came to the crossroads, "Do I go down the path of JAVA or .NET", and as you can see I took the .NET Path.  

In closing this is a viable solution not incurring any extra cost that comes with an Azure Subscription, Azure AD elevated MFA licenses, and of course internal company costs for work with Firewall and Infrastructure teams for on-premise SharePoint Implementations. It does not require a - (SharePoint on-premises single sign-on enabled subscription)

I really hope that you approve and not retire this great piece of MSFT Documentation by Kick Evans and  Yvan Duhamel, which with this PUSH would give everyone a viable solution.

Robert "Moojjoo" Dannelly - RBD Resume - https://goo.gl/BrujGq

**Comment by MSFTTracyP** [comment]
@moojjoo - I built and tested this solution in my own separate environment in parallel to Kirk Evans, actually, and contributed to the troubleshooting of other implementations in the early going. The decision to retire this content is also not a slight to the work of Premier Support, where I spent over a decade. This solution is by a lot of people not listed, and I appreciate all of their work. I also appreciate that you describe yourself as 'an evangelist | developer | engineer | supporter of Microsoft'. I'm glad about that. I'm the same, and also a writer.

That being said, my decision to retire this content was based upon conversations with Kirk Evans, who has already been consulted and weighed in. Considered in the decision was ease of use for customers given our early implementation and Azure advancements in the area, customer feedback on both articles, and internal considerations of no bearing here. While I will bubble up the question of persisting this article again, I don't feel it's right to lead you to believe this article will remain posted in favour of one that customers have reported both easier, and more up-to-date.

Thank you for your feedback. I'll take this response and be sure that the necessary people see your arguments here.

**Comment by moojjoo** [comment]
@MSFTTracyP thank you so much for the thorough and quick response.  I **have to ask the question will this remain a viable solution - #IMPORTANT** or will **Microsoft require customers to use the Gallary and purchase SharePoint on-premises single sign-on enabled subscription**  If the solution is going away I will need to call this out as an impediment in our project ASAP.   

Again, this is critical for the solution we are building for our client in-house as the PROJECT is larger than just getting SharePoint access, but rather putting the power into the business to manage their end users for their applications using Microsoft's wonderful https://developer.microsoft.com/en-us/graph (Crossing fingers that Applications for Graph and Azure AD will make it out of Beta soon). - https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta.

@moojjoo

**Comment by MSFTTracyP** [comment]
@moojjoo - I better understand your question, now. I've alerted people to this thread, and will pose this question to them and let you know the decision that's handed down. Thanks for your persistence on this.

**Comment by mypil** [comment]
@MSFTTracyP - Do you have any updates on this?

**Comment by mypil** [comment]
@MSFTTracyP - I would like to follow up on this. Do you have any updates?

cc: @LauraWi

**Comment by MSFTTracyP** [comment]
@moojjoo - I'm sorry to say that I will be closing this pull request and that the article is, indeed, slated for retirement. I want to thank you for raising these questions that I put forward for debate, not only in the Product Group, but to other involved parties. I don't currently have an exact E.T.A. for the retirement of this article. Please feel free to add comment to it for any other concerns you may have, but it will, at some point in the near future, face retirement in favour of Azure methods for the same process. I'd like to thank you, again, for your efforts on behalf of your customers on this. The article will stand as-is, for the time being.

**Comment by moojjoo** [comment]
MSFTTracyP first and formost I have to know are you a HE or SHE?  Secondly, I am still concerned with the project we are working on.  We have this as a raised risk, but would like to know more from MSFT's side - 

> I have to ask the question will this remain a viable solution - #IMPORTANT or will Microsoft require customers to use the Gallary and purchase SharePoint on-premises single sign-on enabled subscription If the solution is going away I will need to call this out as an impediment in our project ASAP.
> 
> Again, this is critical for the solution we are building for our client in-house as the PROJECT is larger than just getting SharePoint access, but rather putting the power into the business to manage their end users for their applications using Microsoft's wonderful https://developer.microsoft.com/en-us/graph (Crossing fingers that Applications for Graph and Azure AD will make it out of Beta soon). - https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta.

**Comment by LauraWi** [comment]
@moojjoo Your question is inappropriate and irrelevant to the issue at hand. Writers aren't in a position to speak in any official capacity on the viability of solutions.

**Comment by mypil** [comment]
@officedocsbot close

**Pull request closed without merging**